test-data/unit/fixtures/async_await.pyi

```diff
@@ -11,7 +11,7 @@ class int: pass
 class str: pass
 class dict: pass
```

Comment:
The idea is that if you also fix few ``dict``s and ``set``s, then we could turn on the asserts that are now commented out.